cypress.env.example.json

```
@@ -1,14 +1,8 @@
 {
-  "plio_backend": {
-    "convert_social_auth_token_url": "http://0.0.0.0:8001/auth/convert-token",
-    "client_id": "",
-    "client_secret": ""
-  },
-  "auth": {
-    "google": {
-      "refresh_token": "",
-      "client_id": "",
-      "client_secret": ""
-    }
-  }
+  "auth_google_refresh_token": "",
+  "auth_google_client_id": "",
+  "auth_google_client_secret": "",
+  "plio_backend_convert_social_auth_token_url": "http://0.0.0.0:8001/auth/convert-token",
+  "plio_backend_client_id": "",
+  "plio_backend_client_secret": ""
```

maybe we can omit `plio_`?
```suggestion
  "backend_convert_social_auth_token_url": "http://0.0.0.0:8001/auth/convert-token",
  "backend_client_id": "",
  "backend_client_secret": ""
```